lib/lua/lua_init.lua

```diff
@@ -0,0 +1,69 @@
+-- Private table of Rust functions
+local rust = __rust
+__rust = nil
+
+-- Initialize the workspaces
+config.init_workspaces = function(settings)
+    assert(type(settings) == 'table', "settings: expected table")
+    for ix, val in pairs(settings) do
+        assert(type(ix) == 'number', "settings: expected number-indexed array")
+        assert(type(val) == 'table', "settings: expected array of tables")
+
+        val.name = val.name or ""
+        val.mode = val.mode or "tiling"
+    end
+    rust.init_workspaces(settings)
+end
+
+-- Create a new keybinding to register with Rust
+config.key = function(mods, key, action, loop)
+    assert(type(mods) == 'table', "modifiers: expected table")
+    assert(type(key) == 'string', "key: expected string")
+    if loop == nil then loop = true end
+    if type(action) ~= 'string' and type(action) ~= 'function' then
+        error("action: expected string or function", 2)
+    end
+    return {
+        mods = mods, key = key, action = action, loop = loop
+    }
+end
+
+local use_key = ", use the `key` or `config.key` method to create a keybinding"
+
+-- Converts a list of modifiers to a string
+local function keymods_to_string(mods, key)
+    table.insert(mods, key)
+    local turn = table.concat(mods, ',')
+    print(turn)
```

Comment:
You probably should clear out your debug print statements, even if you don't use the function yet.